sidecar/app/routes/start.py

```diff
@@ -47,11 +69,17 @@ def start_ipa_helper(
     multi_threading: Annotated[bool, Form()],
     disable_metrics: Annotated[bool, Form()],
     background_tasks: BackgroundTasks,
+    request: Request,
 ):
     # pylint: disable=too-many-arguments
+    query_manager = request.app.state.QUERY_MANAGER
+    if not query_manager.capacity_available:
+        raise HTTPException(status_code=503, detail="Capacity unavailable")
+
+    settings = get_settings()
     role = settings.role
     if not role or role == role.COORDINATOR:
-        raise Exception("Cannot start helper without helper role.")
+        raise IncorrectRoleError("Cannot start helper without helper role.")
```

Comment:
this message could be made less ambiguous by showing the value of `role` variable on this helper.